lib/simulator-xcode-10.js

```diff
@@ -0,0 +1,36 @@
+import SimulatorXcode93 from './simulator-xcode-9.3';
+import { getAppContainer, appInfo } from 'node-simctl';
+import { fs } from 'appium-support';
+
+
+class SimulatorXcode10 extends SimulatorXcode93 {
+  constructor (udid, xcodeVersion) {
+    super(udid, xcodeVersion);
+  }
+
+  /**
+   * Verify whether the particular application is installed on Simulator.
+   * @override
+   *
+   * @param {string} bundleId - The bundle id of the application to be checked.
+   * @return {boolean} True if the given application is installed.
+   */
+  async isAppInstalled (bundleId) {
+    try {
+      const appContainer = await getAppContainer(this.udid, bundleId, false);
+      const exists = await fs.exists(appContainer);
+      return exists && appContainer.endsWith('.app');
```

Comment:
should we use short calculation here (if appContainer.endsWith('.app') is false then there is no point to check existence)

Reply:
Good idea.